charts/load-test-http/values.yaml

```diff
@@ -28,3 +28,20 @@
 ###   http/latency-mean: 50
 ###   http/latency-p99: 200
 # SLOs: null
+
+### readinessDefinitions   A map of Kubernetes resource kinds to group/version/resource and condition
+### group/version/kind and namespace/name can be used to uniquely identify a resource in a cluster
+### condition is a status condition that indicates the resource is ready for use
+readinessDefinitions:
+  pod:
```

Comment:
`pod` is not something that any of use cases directly involve (deployment; knative; kserve, Seldon, statefulset -- all create pods, but we don't check for the pod readiness in any of these cases).
***
Let us please remove `pod`.

Reply:
Agreed. Pod is not very interesting. Are there other resources we would like to add to this list now?

Reply:
For now, service and deployment seem good.

Reply:
removed